xdsl/dialects/csl/csl_wrapper.py

```diff
@@ -303,6 +311,18 @@ def layout_yield_op(self) -> YieldOp:
         """
         return cast(YieldOp, self.layout_module.block.last_op)
 
+    @property
+    def exported_symbols(self) -> Sequence[BlockArgument]:
+        """
+        Get the exported symbols.
+        """
+        return self.program_module.block.args[
+            2 + len(self.params) + len(self.layout_yield_op.fields) :
+        ]
```

Comment:
Can we get a test for this? Index magic always makes me anxious :sweat_smile:

Reply:
Added as suggested